src/clr-angular/data/datagrid/datagrid-page-size.spec.ts

```diff
@@ -37,7 +37,8 @@ export default function(): void {
         it('displays a select with a default pageSize if no input is given', function() {
           const select = context.clarityElement.querySelector('select');
           expect(select).not.toBeNull();
-          expect(select.innerText).toMatch('0');
+          expect(select.options.length).toBe(1);
```

Comment:
Are these changes related to this PR?

Reply:
No, but the build doesn't pass without it. It's already a part of master branch. I just added those fix for now, but when I rebased it should already be incorporated into it.